docs/src/main/paradox/docs/releases/v1.7-to-v1.8-migration.md

```diff
@@ -0,0 +1,540 @@
+# v1.7 To v1.8 Migration
+
+The v1.8 release of Delta introduces PostgreSQL as sole option for the Delta primary event store.
+
+(To have a complete list of improvements, @ref:[see here](./v1.8-release-notes.md)).
+
+These improvements require to replay all existing events, moving them from Cassandra to PostgreSQL, and to reindex all
+data to Elasticsearch/Blazegraph.
+
+The following steps describe how to perform the migration. At the end, Delta v1.8 will be running using PostgreSQL as
+primary store.
+
+Depending on the number and the complexity of existing resources and schemas in the system, the migration may take
+several hours to complete.
+
+The migration itself can be run in three ways:
+
+* Switching off your Delta 1.7-base Nexus deployment, and re-starting it with Delta 1.8. Then re-uploading all the data
+  in the new instance. This only requires setting up the PostgreSQL instance, and upgrading the Delta image.
+* Switching off Delta 1.7, and starting Delta 1.8 by following the steps described in this document.
+* Starting a Delta 1.8 in parallel to Delta 1.7 by following the steps described in this document. When ready, the 1.7
+  can be switched off and be replaced by Delta 1.8. This step also requires to run a second instance of Blazegraph.
+
+The choice of migration will depend on your current setup, needs, and uptime requirements that you want to uphold for
+your users. At Blue Brain, we have opted to run two instances
+in parallel. This allows for an online migration from Cassandra to Postgres that does not disrupt the users using Delta
```

Comment:
Same, should it be `PostgreSQL` instead of `Postgres` to match the documentation?

Reply:
Corrected